### PR TITLE
1999 AYTQ deploying test, preprod environments via AKS workflow jobs

### PR DESCRIPTION
**Pull request by shaheislamdfe**
### Context

Adding test and preprod environments for aytq

### Changes proposed in this pull request

Adding configuration for Terraform variables, aswell as config values and makefile targets for different environments. GHA workflow added in parallele aswell as AZURE_CREDENTIALS for relevant environments which have been tested.

### Guidance to review

Link to successful workflows below for test and preprod.
https://github.com/DFE-Digital/access-your-teaching-qualifications/actions/runs/11032093796

URLs for Test;
https://access-your-teaching-qualifications-test.test.teacherservices.cloud
https://check-a-teachers-record-test.test.teacherservices.cloud

URLs for Preprod;
https://access-your-teaching-qualifications-preprod.test.teacherservices.cloud
https://check-a-teachers-record-preprod.test.teacherservices.cloud

### Link to Trello card

https://trello.com/c/FBOTpKnP/1999-aytq-deploy-test-environment
https://trello.com/c/YqMPtXPh/2026-aytq-preproduction-environment

### Checklist

- [x] Attach to Trello card
- [x] Rebased main
- [x] Cleaned commit history
- [x] Tested by running locally

### CODE REVIEW

File: global_config/production.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+CONFIG=prod
```
Comment (saliceti) [inline global_config/production.sh]: production

File: global_config/production.sh [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+ENVIRONMENT=production
+CONFIG_SHORT=pd
+AZURE_SUBSCRIPTION=s189-teacher-services-cloud-production
+AZURE_RESOURCE_PREFIX=s189p01
+KV_PURGE_PROTECTION=false
```
Comment (saliceti) [inline global_config/production.sh]: true

File: terraform/application/config/preprod_Terrafile [hunk 3]
```diff
@@ -0,0 +1,3 @@
+aks:
+    source:  "https://github.com/DFE-Digital/terraform-modules"
+    version: "stable"
```
Comment (saliceti) [inline terraform/application/config/preprod_Terrafile]: testing

File: Makefile [hunk 4]
```diff
@@ -190,6 +190,18 @@ aks-review: test-cluster ## Setup review environment for AKS
...
+.PHONY: aks-test
+aks-test: test-cluster
+	$(eval include global_config/test.sh)
+
+.PHONY: aks-preprod
+aks-preprod: test-cluster
+	$(eval include global_config/preprod.sh)
+
+.PHONY: aks-production
+aks-preprod: production-cluster
```
Comment (saliceti) [inline Makefile]: aks-production

**Comment by shaheislamdfe** [comment]
[AYTQ: Deploy test environment](https://trello.com/c/FBOTpKnP/1999-aytq-deploy-test-environment)

**Comment by shaheislamdfe** [comment]
[AYTQ: Deploy test environment](https://trello.com/c/FBOTpKnP/1999-aytq-deploy-test-environment)

**Comment by shaheislamdfe** [comment]
[AYTQ: preproduction environment](https://trello.com/c/YqMPtXPh/2026-aytq-preproduction-environment)

**Review by saliceti — APPROVED** [review]

**Pull request merged**